gradle/libs.versions.toml

```diff
@@ -8,9 +8,9 @@ managed-freemarker = "2.3.32"
 managed-pegdown = "1.6.0"
 managed-evo-inflector = "1.3"
 
-groovy = "4.0.17"
+groovy = "4.0.18"
 kotlin = "1.9.22"
-ksp = "1.9.22-1.0.17"
+ksp = "1.9.22-1.0.18"
```

Comment:
please, use the ksp version exposed by core instead of defining it here.

Reply:
Ok, but how to do it? All projects defining ksp version . Look here https://github.com/micronaut-projects/micronaut-serialization/blob/master/gradle/libs.versions.toml or here https://github.com/micronaut-projects/micronaut-data/blob/master/gradle/libs.versions.toml or here https://github.com/micronaut-projects/micronaut-validation/blob/master/gradle/libs.versions.toml

Reply:
see https://github.com/micronaut-projects/micronaut-openapi/pull/1465